pkg/resources/schema.go

```diff
@@ -145,18 +165,45 @@ func Schema() *schema.Resource {
 	}
 }
 
+func ImportSchema(ctx context.Context, d *schema.ResourceData, meta any) ([]*schema.ResourceData, error) {
+	log.Printf("[DEBUG] Starting schema import")
+	client := meta.(*provider.Context).Client
+	id := helpers.DecodeSnowflakeID(d.Id()).(sdk.DatabaseObjectIdentifier)
+
+	s, err := client.Schemas.ShowByID(ctx, id)
+	if err != nil {
+		return nil, err
+	}
+	if err := d.Set("name", s.Name); err != nil {
+		return nil, err
+	}
+
+	if err := d.Set("database", s.DatabaseName); err != nil {
+		return nil, err
+	}
+
+	if err := d.Set("comment", s.Comment); err != nil {
+		return nil, err
+	}
+
+	if err := d.Set("is_transient", fmt.Sprintf("%t", s.IsTransient())); err != nil {
```

Comment:
I think we already have mapping methods for 3 value bools

Reply:
Done.